hooks/tk-multi-publish2/basic/nuke/publish_files.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import pprint
+import nuke
+import sgtk
+from sgtk.platform import engine
```

Comment:
This should be unnecessary and is not part of the public api. You should be able to get this from: `self.sgtk.shotgun.find_one()`